encoders/numeric/test.py

```diff
@@ -0,0 +1,111 @@
+import os
```

Comment:
This needs to be removed, every executor needs have its own self contained tests and no dependency from other files in the tree